mindsdb/utilities/render/sqlalchemy_render.py

```diff
@@ -76,10 +81,7 @@ def __init__(self, dialect_name):
             # update version for support float cast
             self.dialect.server_version_info = (8, 0, 17)
 
-        self.types_map = {}
-        for type_name in sa_type_names:
-            self.types_map[type_name.upper()] = getattr(sa.types, type_name)
-        self.types_map['BOOL'] = self.types_map['BOOLEAN']
+        self.types_map = types_map
```

Comment:
it is used only once in line 366
```
        type = self.types_map[typename]
```
maybe replace it to  `type = types_map[typename]`?